data/categories.txt

```diff
@@ -4763,6 +4763,318 @@ vi:Nhà vật lý trị liệu
 zh-Hans:物理治疗师
 zh-Hant:物理治療師
 
+healthcare-alternative
+en:alternative medicine
+ar:الطب البديل
+be:Альтэрнатыўная медыцына
+bg:Алтернативна медицина
+cs:Alternativní medicína
+da:Alternativ medicin
+de:Alternative Medizin
+el:Εναλλακτική ιατρική
+es:Medicina alternativa
+et:Alternatiivne meditsiin
+eu:Medikuntza alternatiboa
+fa:طب جایگزین
+fi:Vaihtoehtoinen lääketiede
+fr:Médecine alternative
+he:רפואה אלטרנטיבית
+hu:Alternatív gyógyászat
+id:Pengobatan alternatif
+it:Medicina alternativa
+ja:代替医療
+ko:대체 약품
+mr:पर्यायी औषध
+nb:Alternativ medisin
+nl:Alternatieve geneeskunde
+pl:Medycyna alternatywna
+pt:Medicina alternativa
+pt-BR:Medicina alternativa
+ro:Medicină alternativă
+ru:Альтернативная медицина
+sk:Alternatívna medicína
+sv:Alternativ medicin
+sw:Dawa mbadala
+th:การแพทย์ทางเลือก
+tr:Alternatif tıp
+uk:Альтернативна медицина
+vi:Liều thuốc thay thế
+zh-Hans:替代药物
+zh-Hant:替代藥物
+
+healthcare-audiologist
+en:Audiology|Audiologist
+ar:السمع|السمعيات
+be:Аўдыялогія|Аўдыёлаг
+bg:Аудиология|Аудиолог
+cs:Audiologie|Audiolog
+da:Audiologi|Audiolog
+de:Audiologie|Audiologin|Audiologe
+el:Ακοολογία|Ακουολόγος
+es:Audiología|Audiólogo
+et:Audioloogia|Audioloog
+eu:Audiologia|Audiologoa
+fa:شنوایی شناس|شنوایی شناسی
+fi:Audiologia|Audiologi
+fr:Audiologie|Audiologiste
+he:אודיולוג|אודיולוגיה
+hu:Audiológia|Audiológus
+id:Audiologi|Audiologis
+it:Audiologia|Audiologo
+ja:オーディオロジー|オーディオロジスト
+ko:청력학|청력학자
+mr:ऑडिओलॉजी|ऑडिओलॉजिस्ट
+nb:Audiologi|Audiolog
+nl:Audiologie|Audioloog
+pl:Audiologia|Audiolog
+pt:Audiologia|Audiologista
+pt-BR:Fonoaudiologia|Fonoaudiólogo
+ro:Audiologie|Audiolog
+ru:Аудиология|Аудиолог
+sk:Audiológia|Audiológ
+sv:Audiologi|Audiolog
+sw:Mtaalamu wa kusikia|Audiology
+th:โสตวิทยา|นักโสตสัมผัสวิทยา
+tr:Odyoloji|Odyolog
+uk:Аудіологія|Аудіолог
+vi:Thính học
+zh-Hans:听力学|听力学家
+zh-Hant:聽力學|聽力學家
+
+healthcare-blood_donation
```

Comment:
Please, comment/remove this category or make simple "Blood"
Due to search specific it will interfere with other categories (by tokens "center").